### ISSUE TITLE
[spec] Example for `aalt` feature is not well-formed

### ISSUE THREAD

**Issue by brawer** [adobe-type-tools/afdko#91]
In section [8.a, The all alternates (‘aalt’) feature](http://www.adobe.com/devnet/opentype/afdko/topic_feature_file_syntax.html#8.a), the OpenType Feature File specification currently gives an example that is syntactically not well-formed. When passing the spec snippet to `makeotf`, the tool (correctly) rejects the code.

Current example feature file:
```
  feature aalt {...} aalt;
  feature scmp {...} scmp;
  feature salt {
     sub a from [a.alt1 a.alt2 a.alt2];
     sub e [c d e] f by [c.mid d.mid e.mid];
     sub b by b.alt;
  } salt;
```

Instead, the middle line of `feature salt` should have an apostrophe; also, the duplicate `a.alt2` should probably be `a.alt3`:
```
  feature aalt {...} aalt;
  feature scmp {...} scmp;
  feature salt {
     sub a from [a.alt1 a.alt2 a.alt3];
     sub e [c d e]' f by [c.mid d.mid e.mid];
     sub b by b.alt;
  } salt;
```

**Comment by readroberts** [comment]
I just pushed the fix for this in v1.1.7 of the file.

**Comment by brawer** [comment]
Thank you!
